facturacion_mexico/facturacion_fiscal/tests/test_nota_credito_descuento_relacion_01.py

```
@@ -266,62 +430,181 @@ def test_sin_items(self):
-			patch(
-				"facturacion_mexico.facturacion_fiscal.utils.get_cuenta_descuentos",
-				return_value=CUENTA_DESC,
-			),
-			patch(
-				"facturacion_mexico.facturacion_fiscal.utils.credit_note_lines_use_discount_account",
-				return_value=True,
-			),
-		):
-			inst._detect_nota_credito_motivo()
-		self.assertEqual(inst.fm_tipo_nota_credito, "Descuento / Bonificación")
+	def _derivar(self, ffm, si, origen, cuenta):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_derivar`

Add return type annotation: `None`

(ANN202)